src/app/static/src/app/generated.d.ts

```diff
@@ -109,6 +112,10 @@ export interface CalibrateResultResponse {
     upper: number | null;
     [k: string]: any;
   }[];
+  uploadMetadata: {
+    outputZip: UploadMetadataElements
+    outputSummary: UploadMetadataElements
+  } | null
```

Comment:
Was this change done manually or via `generate-types.sh`? See message at top of the file... manual changes will be overwritten.